docs/usage.rst

```diff
@@ -105,9 +106,21 @@ To validate a RAML file with Python:
    >>> validate(RAML_FILE)
    InvalidRootNodeError: RAML File does not define an API title.
 
+.. code-block:: python
+
+   >>> from ramlfications import validate
+   >>> RAML_FILE = "/path/to/invalid/no-base-uri-no-title.raml"
+   >>> validate(RAML_FILE)
+   MultipleValidationErrors:
```

Comment:
At the end of `parse_raml` in `parser.py`, I'm only raising a `MultipleValidationErrors` exception if there are more than one:

    if root.errors:
        if len(root.errors) == 1:
            raise root.errors[0]
        else:
            raise MultipleValidationErrors(root.errors)